dspace-api/src/main/java/org/dspace/app/sherpa/submit/SHERPASubmitService.java

```diff
@@ -37,12 +38,28 @@ public void setSherpaService(SHERPAService sherpaService) {
         this.sherpaService = sherpaService;
     }
 
-    public SHERPAResponse searchRelatedJournals(Context context, Item item) {
+    public List<SHERPAResponse> searchRelatedJournals(Context context, Item item) {
         Set<String> issns = getISSNs(context, item);
         if (issns == null || issns.size() == 0) {
             return null;
         } else {
-            return sherpaService.searchByJournalISSN(StringUtils.join(issns, ","));
+            // SHERPA v2 API no longer supports "OR'd" ISSN search, perform individual searches instead
+            Iterator<String> issnIterator = issns.iterator();
+            List<SHERPAResponse> responses = new LinkedList<>();
+            while (issnIterator.hasNext()) {
+                String issn = issnIterator.next();
+                SHERPAResponse response = sherpaService.searchByJournalISSN(issn);
+                if (response.isError()) {
+                    // Continue with loop
+                    log.warn("Failed to look up SHERPA ROMeO result for ISSN: " + issn);
+                }
+                // Store this response, even if it has an error (useful for UI reporting)
+                responses.add(response);
+            }
+            if (responses.isEmpty()) {
+                responses.add(new SHERPAResponse("SHERPA ROMeO lookup failed"));
```

Comment:
Did the lookup "fail", it could be that there are no responses ?

Reply:
Yes - in this case "responses" is a linked list of each individual API response objects, not the list of items included in the JSON response. (because we're chaining some queries for multiple ISSNs). 
Even an "empty" API response should still be represented in a valid SHERPAResponse object.

Reply:
You are correct, resolving.